torch/nn/quantized/modules/conv.py

```diff
@@ -317,27 +339,24 @@ def _get_name(self):
 
     def set_weight_bias(self, w, b):
         # type: (torch.Tensor, Optional[torch.Tensor]) -> None
-        self._packed_params = torch.ops.quantized.conv3d_prepack(
-            w, b, self.stride, self.padding, self.dilation, self.groups)
+        self._packed_params.set_weight_bias(w, b)
 
     def _weight_bias(self):
-        return torch.ops.quantized.conv3d_unpack(self._packed_params)
+        return self._packed_params._weight_bias()
```

Comment:
this is conv2d_unpack....we'll need to have Conv3dPackedParams module for this..

Reply:
Are there any tests or checks for this? It all passes just fine in the test suite....

Reply:
cc @dskhudia @BIT-silence do we have tests for conv3d_prepack and conv3d_unpack?

Reply:
@jerryzh168 figured out the missing test coverage, please see https://github.com/pytorch/pytorch/pull/30247

Reply:
> cc @dskhudia @BIT-silence do we have tests for conv3d_prepack and conv3d_unpack?

Yes. There is.